pkg/codegen/docs/templates/package_details.tmpl

```diff
@@ -2,7 +2,7 @@
 <h2 id="package-details">Package Details</h2>
 <dl class="package-details">
 	<dt>Repository</dt>
-	<dd><a href="{{ htmlSafe .Repository }}">{{ htmlSafe .Repository }}</a></dd>
+	<dd><a href="{{ htmlSafe .Repository }}">{{ htmlSafe .DisplayName }} {{ htmlSafe .RepositoryName }}</a></dd>
```

Comment:
Do we want the space between `{{ htmlSafe .DisplayName }}` and `{{ htmlSafe .RepositoryName }}`? It's a very large diff in our generated output, and I don't think it's intentional.

Reply:
this is intentional yes, if you look at the screenshot this is the separation between the name of the package and the path to the repo